setup.py

```diff
@@ -13,10 +14,12 @@
     license='BSD 3-clause',
     url='http://github.com/tensorlib/tensorlib/',
     install_requires=['numpy',
-                      'scipy',],
+                      'scipy', ],
     classifiers=['Development Status :: 3 - Alpha',
                  'Intended Audience :: Science/Research',
                  'License :: OSI Approved :: BSD License',
                  'Operating System :: OS Independent',
                  'Topic :: Scientific/Engineering'],
+    package_data={'tensorlib': [op.join('datasets', 'data', '*.txt'),
+                                op.join('datasets', 'data', '*.mat')]}
```

Comment:
Nitpick for consistency: Could you either change all occurences of `os.path` to `op` and delete the import of `os` or remove `import os.path as op` and replace all `op` by `os.path`? I would prefer the latter for explicitness since we don't have that many occurences.